.github/workflows/server_checks.yml

```diff
@@ -199,7 +199,7 @@ jobs:
 
   check_workflow_runs:
     needs: doc-build
-    uses: ansys/grantami-bomanalytics/.github/workflows/check-concurrent-workflows.yml@main
+    uses: ansys/pygranta/.github/workflows/check-concurrent-workflows.yml@main
```

Comment:
Can you add a name here? I noticed in the PR in the PyGranta repo that this just has the "check_workflow_runs" ID, instead of an actual name.